pkg/controller/nodenetworkconfigurationpolicy/selectors/manager_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package selectors
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestUnit(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Policy Node Selectors Test Suite")
```

Comment:
Let's keep this generic (since the package name is generic) and make it Policy Selectors Test Suite